exercises/bank-account/BankAccount.cs

```diff
@@ -12,15 +12,15 @@ public void Close()
         throw new NotImplementedException("You need to implement this function.");
     }
 
-    public float Balance
+    public decimal Balance
```

Comment:
Nice! Could you also update the `Example.cs` implementation (which is in the same directory)?

Reply:
Ok, sure! Didn't see that